applications/StructuralMechanicsApplication/structural_mechanics_application.cpp

```diff
@@ -342,6 +342,8 @@ void KratosStructuralMechanicsApplication::Register() {
 
     // Strain measures
     KRATOS_REGISTER_VARIABLE(HENCKY_STRAIN_VECTOR);
+    KRATOS_REGISTER_VARIABLE(INITIAL_STRAIN_VECTOR);
```

Comment:
I already added this to core/imp-def, it was defined on the core...

Reply:
I agree that it makes sense to have these ones in the core.